guides/source/command_line.md

```diff
@@ -102,8 +102,8 @@ Please choose a generator below.
 
 Rails:
   assets
+  channel
   controller
-  generator
```

Comment:
Why generator was removed? It is there for sure. https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/generator/generator_generator.rb